migrations/0007_issuers_v3.up.sql

```diff
@@ -0,0 +1,58 @@
+-- v3_issuer - new issuer structure
+CREATE TABLE v3_issuers(
+    issuer_id uuid primary key default uuid_generate_v4(),
+    issuer_type text not null,
+    created_at timestamp not null default now(),
+    expires_at timestamp,
+    last_rotated_at timestamp,
+    valid_from timestamp not null default now(),
+    buffer integer not null default 1,
+    days_out integer not null default 1,
+    overlap integer not null default 0,
+    issuer_cohort integer not null default 1,
+    redemption_repository text not null default 'dynamodb',
+    version integer default 3,
+    max_tokens integer default 40,
+    duration text default null
+);
+
+-- v3_issuer_keys - holds all sign/redeem keys for issuers v3
+CREATE TABLE v3_issuer_keys (
+    key_id uuid primary key default uuid_generate_v4(),
+    issuer_id uuid references v3_issuers(issuer_id),
+    created_at timestamp not null default now(),
+    start_at timestamp,
+    end_at timestamp,
+    signing_key text not null,
+    public_key text,
+    cohort smallint not null default 1
```

Comment:
could this lead to issues with cohort being small int and v3_issuers.issuer_cohort being int. Also, in the client request code cohort is an int and I dont see any checks to ensure this will not be out of range before a db insert is attempted.

Reply:
will convert issuer cohort in code to int16 to match.